pkgs/development/python-modules/neuralcoref/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, spacy
+, cython
+, pytest
+, boto3
+}:
+
+buildPythonPackage rec {
+  pname = "neuralcoref";
+  version = "4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "23aee7418edf5c2e90b3f9f15931563b07ded8c59de386ae654b403322430ad9";
+  };
+
+  propagatedBuildInputs = [
+    spacy
+    cython
+    pytest
+    boto3
+  ];
+
+  checkInputs = [
+    pytest
+  ];
+
+  doCheck = false;
+  # checkPhase = ''
+  #   ${python.interpreter} -m pytest spacy/tests --vectors --models --slow
+  # '';
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.